PowerEditor/src/WinControls/WindowsDlg/WindowsDlg.cpp

```diff
@@ -946,6 +946,118 @@ void WindowsDlg::doCount()
 	SetWindowText(_hSelf,msg.c_str());
 }
 
+void WindowsDlg::doSort()
+{
+	size_t count = (_pTab != NULL) ? _pTab->nbItem() : 0;
+	auto currrentTabIndex = _pTab->getCurrentTabIndex();
+	NMWINDLG nmdlg;

```

Comment:
You can also use uniform initialization to avoid uninitialized members. Like this: `nmdlg = {};`